react/utils/client/links/uriSwitchLink.ts

```diff
@@ -50,11 +50,14 @@ const extractHints = (query: ASTNode, meta) => {
 export const createUriSwitchLink = (baseURI: string, workspace: string) =>
   new ApolloLink((operation: Operation, forward?: NextLink) => {
     operation.setContext(({ fetchOptions = {}, runtime: {appsEtag, cacheHints} } : OperationContext) => {
+      const oldContext = operation.getContext()
```

Comment:
**nit**: the old context is already available as `setContext` param. I think something like this would be clearer:

```
operation.setContext((oldContext: OperationContext) => {
  const {fetchOptions = {}, runtime: {appsEtag, cacheHints}} = oldContext
  const oldMethod = fetchOptions.method || 'POST'
  ...
  ...
})
```

Reply:
thanks @brunoabreu !!